src/components/Modal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Modal as AntdModal } from "antd";
+import { H2 } from "@leafygreen-ui/typography";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+export interface ModalProps {
+  footer: JSX.Element;
+  title: string;
+  "data-cy": string;
+  visible: boolean;
+  onCancel: (e?: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+}
+
+export const Modal: React.FC<ModalProps> = ({
+  footer,
+  title,
+  onCancel,
+  visible,
+  children,
+  "data-cy": dataCy,
+}) => (
+  <StyledModal
+    maskStyle={{
+      backgroundColor: "hsla(0, 100%, 0%, .9)",
```

Comment:
You can use the leafygreen black and an opacity for this like we use in. https://github.com/evergreen-ci/spruce/blob/master/src/components/WelcomeModal/WelcomeModal.tsx#L91

Reply:
i used hsla instead of color because setting the opacity through the opacity css property resulted in a flicker during the modal open animation
![hsla](https://user-images.githubusercontent.com/10734386/85768785-62d44300-b6e7-11ea-8fab-67097a2971e9.gif)

![flicker](https://user-images.githubusercontent.com/10734386/85768764-5d76f880-b6e7-11ea-9ad9-97a67b661214.gif)

Reply:
just update the color because i see they dont match